app/views/ctc/questions/life_situations2020/edit.html.erb

```diff
@@ -0,0 +1,23 @@
+<% @main_question = t("views.ctc.questions.life_situations_2020.title") %>
+
+<% content_for :page_title, @main_question %>
+
+<% content_for :form_card do %>
+  <h1 class="h2"><%= @main_question %></h1>
+
+  <p class="spacing-below-15"><%= t("views.ctc.questions.life_situations_2020.p1") %></p>
+
+  <%= form_with model: @form, url: current_path, local: true, method: "put", builder: VitaMinFormBuilder do |f| %>
+    <div class="form-card__stacked-checkboxes spacing-above-0 spacing-below-15">
+      <%= f.cfa_checkbox(:cannot_claim_me_as_a_dependent, t("views.ctc.questions.life_situations_2020.cannot_claim_me_as_a_dependent"), options: { checked_value: "yes", unchecked_value: "no" }) %>
+      <%= f.cfa_checkbox(:member_of_the_armed_forces, t("views.ctc.questions.life_situations_2020.member_of_the_armed_forces"), options: { checked_value: "yes", unchecked_value: "no" }) %>
+    </div>
+    <div class="reveal" style="max-width: 47.5rem;">
```

Comment:
I set this `max-width` to a magic number, but It looked weird for the reveal container not be the same size of the checkboxes on larger screens.

Reply:
agreed. the varying widths of different components really frustrates me. I wish we could set it up so that everything by default takes up the size of its container, and then make things smaller if necessary.

Reply:
Yes and agreed, I would love to migrate to a "full width" of whatever container approach.